.github/workflows/test.yml

```diff
@@ -8,20 +8,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04, ubuntu-20.04, ubuntu-22.04, macos-11, macos-12]
+        os: [ubuntu-20.04, ubuntu-22.04, macos-11, macos-12]
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
       - name: Install ubuntu dependencies
         if: ${{ matrix.os == 'ubuntu-*' }}
         run: sudo apt update && sudo apt install build-essential cmake
-      - name: Build svZeroDSolver
-        run: |
-          mkdir Release
-          cd Release
-          cmake -DCMAKE_BUILD_TYPE=Release ..
-          cmake --build .
-      - name: Install test dependencies
+      - name: Install svZeroDPlus
```

Comment:
I'm having trouble running pytest when I build using CMake on Sherlock. The problem is that pytest cannot find the svzerodplus module when importing it in test_integration_cpp.py. Suggestions?

Reply:
See below